KVKCalendar/Classes/Style.swift

```diff
@@ -85,6 +85,7 @@ public struct TimelineStyle {
     public var timeColor: UIColor = .systemGray
     public var timeFont: UIFont = .systemFont(ofSize: 12)
     public var scrollToCurrentHour: Bool = true
```

Comment:
This is the same property to disable/enable auto scroll 🙂

Reply:
I've tryed it and actually, it scrolls to top if this property is set to false. (I comment the line that is responsible for this behaviour :p )